python/graphscope/nx/classes/graph.py

```diff
@@ -2058,3 +2087,22 @@ def _project_to_simple(self, v_prop=None, e_prop=None):
         graph._op = op
         graph._is_client_view = False
         return graph
+
+    def _check_converted(self):
```

Comment:
what is this function for?
the name and its functioning are TOTALLY inconsistent!

Reply:
revise to `_arrow_to_dynamic`